docs/notebooks/test_snia.ipynb

```diff
@@ -20,10 +20,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    opsim_name = Path(tdastro._TDASTRO_TEST_DATA_DIR, "opsim_shorten.db")
</pre><p>_TDASTRO_TEST_DATA_DIR is already defined as a Path object in src/tdastro/__init__.py, so this can be simplified to </p><p> </p><p>tdastro._TDASTRO_TEST_DATA_DIR / "opsim_shorten.db"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed